templates/nvidia.yaml

```diff
@@ -9,17 +9,14 @@ metadata:
     kubeaddons.mesosphere.io/provides: nvidia
   annotations:
     appversion.kubeaddons.mesosphere.io/nvidia: "0.1"
+    values.chart.helm.kubeaddons.mesosphere.io/nvidia: "https://raw.githubusercontent.com/mesosphere/charts/869a26a8d4061d5b407cd260eac2eee9149f2823/staging/nvidia/values.yaml"
 spec:
   kubernetes:
     minSupportedVersion: v1.15.0
   namespace: kube-system
   cloudProvider:
     - name: aws
       enabled: false
-    - name: docker
-      enabled: false
-    - name: none
```

Comment:
Just curious about the context: Why wont this work in on-prem? Could we potentially add some comments to this section so that the reader can at-a-glance quickly understand the reasons why we expect AWS only use for this addon?

Reply:
nope, you're right this makes sense in on-prem. I removed more than I should

Reply:
ok cool, np!